appeals/web/src/server/appeals/appeal-details/interested-party-comments/interested-party-comments.controller.js

```diff
@@ -82,3 +83,60 @@ export async function renderViewInterestedPartyComment(request, response) {
 		pageContent
 	});
 }
+
+/**
+ * @param {import('@pins/express/types/express.js').Request} request
+ * @param {import('@pins/express/types/express.js').RenderedResponse<any, any, Number>} response
+ */
+export async function renderReviewInterestedPartyComment(request, response) {
+	const { errors, currentComment } = request;
+
+	console.log('renderReviewInterestedPartyComment');
+	console.log(request.currentComment);
+
+	if (!currentComment) {
+		return response.status(404).render('app/404.njk');
+	}
+
+	const pageContent = reviewInterestedPartyCommentPage(request.currentAppeal, currentComment);
+
+	return response.status(200).render('patterns/change-page.pattern.njk', {
+		errors,
+		pageContent
+	});
+}
+
+/**
+ * @param {import('@pins/express/types/express.js').Request} request
+ * @param {import('@pins/express/types/express.js').RenderedResponse<any, any, Number>} response
+ */
+export const renderPostReviewInterestedPartyComment = async (request, response) => {
+	try {
+		const { appealId, commentId } = request.params;
```

Comment:
Just a suggestion, this could be shortened to `const { errors, currentAppeal, params: { appealId, commentId } } = request;`